src/patternfly/components/Popover/docs/code.md

```diff
@@ -18,13 +18,13 @@ A popover is used to provide contextual information for another component on cli
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
-| `.pf-c-popover` | `<div>` |  Creates a popover. |
-| `.pf-c-popover__arrow` | `.pf-c-popover` |  Creates an arrow pointing towards the element the popover describes. ** Required.** |
-| `.pf-c-popover__content` | `.pf-c-popover` |  Creates the body of the popover. ** Required.** |
-| `.pf-c-popover__close` | `.pf-c-popover__content` |  Contains and positions the close icon in the top-right corner of the popover. ** Required.** |
-| `.pf-c-popover__header` | `.pf-c-popover__content` |  The header text area of the popover. |
-| `.pf-c-popover__title` | `.pf-c-popover__header` |  The actual popover title. |
-| `.pf-c-popover__body` | `.pf-c-popover__content` |  The popover's body text. ** Required.** |
+| `.pf-c-popover` | `<div>` |  Creates a popover. Always use it with a modifier class. **Required** |
```

Comment:
Do you think consumers will understand that "Always use it with a modifier class" means to use a class like `.pf-m-left`? I wonder if just updating the description for the modifier classes to something like "A modifier class that puts the arrow to the left of the popover's content" would help connect those dots.

Reply:
I'll add that, I don't think it will hurt and will add more context. Just so I'm sure what you're suggesting, instead of saying "Always use it with a modifier class", you're suggesting something like "Always use it with a modifier class that positions the popover relative to the element that triggered it."

Glad you mentioned this because currently these modifier descriptions are wrong (for example, `pf-m-left` positions the popover to the left of the element that triggered it, so the arrow is actually on the right), and I don't think we should describe what the modifier does by where the arrow is, and instead describe it as a class that positions the popover. I made that change in the tooltip component and forgot to make it here.

Reply:
Oh and I meant to mention that the "always use with a modifier class" is something existing that we use on title, alert, button, and badge, too. Do you think it needs to be updated in those components as well?

Reply:
I was thinking that we would call out the classes that were modifier classes. So anytime we use a modifier class, we start the description by saying "A modifier class that...". 

Class | Applied To | Outcome
-- | -- | --
.pf-m-left | .pf-c-popover | A modifier class that positions the popover to the left of the popover trigger |

But now that you described what you were thinking, I think either could work.

As for making this change in other component examples, I like having consistency. Do you want to update this example and see what everyone thinks?

Reply:
Ah, I see. Personally I think it's fine to not start modifier classes with "A modifier class that...", which assumes that folks understand that `.pf-m-` denotes a class that modifies something in the component. I think it might be repetitive to have each modifier class start with that, too. Personally I think "always use with a modifier class" is explanatory enough, too - it means always use one of the `.pf-m-` classes in the component... but I could be making assumptions, and having the additional descriptions could be helpful :) Agreed about consistency.